tests/func/test_metrics.py

```diff
@@ -1005,3 +1005,13 @@ def test_metrics_diff_deleted_metric(tmp_dir, scm, dvc):
             "a.b.e": {"old": "3", "new": None},
         }
     }
+
+
+def test_metrics_without_scm(tmp_dir, dvc):
+    metrics = {"acc": 0.97, "recall": 0.95}
+    metrics_name = "metrics.json"
+    tmp_dir.gen({metrics_name: json.dumps(metrics)})
+    dvc.add(metrics_name)
+    dvc.metrics.add(metrics_name)
+    with pytest.raises(DvcException, match=r"Only supported for Git repos"):
```

Comment:
I guess we could
```suggestion
    with pytest.raises(NoSCMError):
```
?